mono/utils/mono-threads-windows.c

```diff
@@ -319,6 +319,12 @@ mono_threads_platform_open_thread_handle (HANDLE handle, MonoNativeThreadId tid)
 	return OpenThread (THREAD_ALL_ACCESS, TRUE, tid);
 }
 
+void
+mono_threads_platform_close_thread_handle (HANDLE handle)
+{
+	CloseHandle (handle);
```

Comment:
Not any worse than the old code, but we should be checking return code from all Win32 APIs and asserting/logging IMO.